client/client.go

```diff
@@ -16,6 +18,24 @@ import (
 	"golang.org/x/crypto/ssh/agent"
 )
 
+// InstallPublicFiles installs the public part of the cert and key.
+func InstallPublicFiles(prefix string, cert *ssh.Certificate, pub ssh.PublicKey) error {
```

Comment:
I like `Save` more, yes. Was just trying to make them match. As for `InstallCert` I can't think of a better term. `UploadCertToAgent` maybe but then you'd have to port this all to Java and isn't there enough horribleness in the world?